src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -128,35 +138,46 @@ export const PublishingAccordion = ({
 
   const firstErrorTab = Math.max(getFirstErrorTab(tabErrors), 0);
 
-  const onClickPublish = async () => {
-    setIsLoading(LoadingState.CreatePublishingRequest);
-    const createPublishingRequestTicketResponse = await createTicket(registration.id, 'PublishingRequest');
-    if (isErrorStatus(createPublishingRequestTicketResponse.status)) {
-      dispatch(
-        setNotification({
-          message: t('feedback.error.create_publishing_request'),
-          variant: 'error',
-        })
-      );
-    } else if (isSuccessStatus(createPublishingRequestTicketResponse.status)) {
-      userCanPublish
-        ? dispatch(
-            setNotification({
-              message: t('feedback.success.publish_as_curator'),
-              variant: 'success',
-            })
-          )
-        : dispatch(
-            setNotification({
-              message: t('feedback.success.create_publishing_request'),
-              variant: 'success',
-            })
-          );
-      refetchData();
+  const onClickPublish = async (showDuplicateWarningIgnored = false) => {
```

Comment:
Ser ut som denne gjør flere ting nå, som enten å bare vise duplikat-advarsel, eller å faktisk publisere. Gir det mening å splitte opp i to funksjoner som bare gjør en ting? 

Om jeg skjønner riktig så vil denne funksjonen først kalles med `false` om det finnes en duplikat, og den vil bare gjøre første if, så vil den kalles igjen med `true` etterpå om bruker bekrefter. Virker som det hadde forenklet ting om det var 2 ulike funksjoner som bare gjør en ting

Reply:
Bra innspill. Fikser.